samples/net/sockets/echo_client/src/tcp.c

```diff
@@ -91,6 +91,44 @@ static int start_tcp_proto(struct data *data, struct sockaddr *addr,
 		return -errno;
 	}
 
+#if defined(CONFIG_SOCKS)
+#define SOCKS5_PROXY_V6_ADDR CONFIG_NET_CONFIG_PEER_IPV6_ADDR
```

Comment:
Can we please move these defines to the top of the file? (With a comment that they are used only for CONFIG_SOCKS).